### ISSUE TITLE
Does not detect license header

### ISSUE THREAD

**Issue by jimbojw** [nikku/eslint-plugin-license-header#19]
### Describe the Bug

Attempted to use simple Apache-2.0 SPDX license header. It consistently complains that files that have the header are missing it.

### Steps to Reproduce

This minimal repro uses the `eslint.config.mjs` file itself as the test.

Create a new directory.

```
mkdir tmp
cd tmp
```

Initialize npm.

```
npm init -y
```

Install dependencies:

```
npm install @eslint/js eslint eslint-plugin-license-header
```

Create an `eslint.config.mjs` file with these contents:

```mjs
/**
 * @license SPDX-License-Identifier: Apache-2.0
 */
/**
 * @fileoverview ESLint configuration.
 */

// @ts-check

import eslint from "@eslint/js";
import licenseHeader from "eslint-plugin-license-header";

export default [
  {
    files: ["**/*.mjs"],
  },
  eslint.configs.recommended,
  {
    plugins: {
      "license-header": licenseHeader,
    },
    rules: {
      "license-header/header": [
        "error",
        [
          "/**",
          " * @license SPDX-License-Identifier: Apache-2.0",
          " */"
        ],
      ],
    },
  },
];
```

Run eslint:

```
npx eslint
```

Observe that ESLint complains that the `eslint.config.mjs` file itself is missing the header.

```
/path/to/tmp/eslint.config.mjs
  1:1  error  Missing license header  license-header/header

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

### Expected Behavior

File should be correctly identified as having the SPDX license header.

### Environment

Relevant dependency versions tested:

* @eslint/js: 9.20.0
* eslint: 9.20.1
* eslint-plugin-license-header: 0.6.1

### Notes

Using `npx eslint --fix` inserts numerous copies of the header (with newlines between them). But even with these numerous inserted headers, `npx eslint` still detects the file as missing the header.

**Comment by nikku** [comment]
Good catch, will be closed via https://github.com/nikku/eslint-plugin-license-header/pull/20.

**Comment by nikku** [comment]
Released [`v0.7.0`](https://github.com/nikku/eslint-plugin-license-header/releases/tag/v0.7.0) with the fix.

**Comment by nikku** [comment]
@acuarica Happy to take a PR to relax the `@license` check to also test for `SPDX-License-Identifier`

**Comment by acuarica** [comment]
Glad to hear that. PR sent https://github.com/nikku/eslint-plugin-license-header/pull/21.
